flink-runtime/src/main/java/org/apache/flink/runtime/io/network/ConnectionManager.java

```diff
@@ -26,6 +26,11 @@
  */
 public interface ConnectionManager {
 
+	/**
+	 * Starts the internal related components for network connection and communication.
+	 *
+	 * @return the data port of server bind address if have.
+	 */
```

Comment:
I would write it like this:
```
/**
   * Starts the internal related components for network connection and communication.
   *
   * @return a port to connect to the task executor for shuffle data exchange, -1 if only local connection is possible.
   */
```
I think the same comment is relevant for `NetworkEnvironment`.
We could already squash this fixup into the previous commit.